cli/src/main/java/com/box/l10n/mojito/cli/command/extractioncheck/ExtractionCheckNotificationSenderGithub.java

```diff
@@ -0,0 +1,128 @@
+package com.box.l10n.mojito.cli.command.extractioncheck;
+
+import com.box.l10n.mojito.cli.command.checks.CliCheckResult;
+import com.box.l10n.mojito.github.GithubClients;
+import com.box.l10n.mojito.thirdpartynotification.github.GithubIcon;
+import com.google.common.base.Strings;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.kohsuke.github.GHCommitState;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Configurable;
+
+@Configurable
```

Comment:
brushed on a new pattern that avoids using Aspectj for DI and pass dependencies by ctor instead. Maybe worth a refactoring this older code at some point. 

that code initialize either by CLI args or fully from springboot configuration

https://github.com/box/mojito/blob/f6f74037b984110563db7ea0ae58369cfa029278/cli/src/main/java/com/box/l10n/mojito/cli/command/ExtractionDiffNotificationCommand.java#L228